src/gc/softwarewritewatch.h

```diff
@@ -204,9 +199,13 @@ inline void SoftwareWriteWatch::EnableForGCHeap()
 
     VerifyCreated();
     assert(!IsEnabledForGCHeap());
+    g_gc_sw_ww_enabled_for_gc_heap = true;
 
-    g_sw_ww_enabled_for_gc_heap = true;
-    SwitchToWriteWatchBarrier(true);
+    WriteBarrierParameters args = {};
```

Comment:
I am sorry, I did not know that this is really a C99 feature and not a C++ feature.  So please ignore my comment.